src/main/java/io/stargate/sgv2/jsonapi/service/resolver/model/impl/CreateCollectionCommandResolver.java

```diff
@@ -179,6 +179,18 @@ public static String generateComment(
     return tableCommentNode.toString();
   }
 
+  /**
+   * Validates the vector search options provided in a create collection command. It checks if
+   * vector search is enabled globally, and validates the specific vectorization service
+   * configuration provided by the user. It also ensures the specified vector dimension complies
+   * with system limits.
```

Comment:
Does system limit mean Cassandra maximum, or something else?

Reply:
change to "config limits"